src/theme.js

```diff
@@ -22,10 +22,11 @@ export default {
 		light: '#ccc',
 		link: '#8667a8',
 		hover: '#ed9dc5',
-		focus: '#ed9dc599',
```

Comment:
Any reason for this change?

Reply:
OK, let's use rgba then ;-)

Reply:
The support for `#rrggbbaa` is still limited: https://caniuse.com/#feat=css-rrggbbaa so I'd opt for using `rgba` or polished helpers here.

Saying that, I've submitted an issue: https://github.com/styled-components/polished/issues/306 and might work on the fix.